tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/Program.cs

```diff
@@ -0,0 +1,40 @@
+using System.Diagnostics;
+using GitHubTeamUserStore.Constants;
+
+namespace GitHubTeamUserStore
+{
+    internal class Program
+    {
+        static async Task Main(string[] args)
```

Comment:
Where is the pipeline that we will be running this tool in?

Reply:
I think it is to-be-created per PR description:

> I've also created the variable group, azuresdkartifacts azure-sdk-write-teams variables, which will be used in the pipeline when I create it, after this PR has been merged.

Reply:
@weshaggard and @konrad-jamrozik 
I need to get this merged prior to creating the pipeline. Unlike other tools, this won't be being published to the NET dev feed because it's unusable outside of pipeline so there's really no point. The only test that makes sense here is to pull back what was just stored and ensure it matches the dictionary created from the team/user data and this done immediately after the call to store it. The pipeline just needs to build and run the tool to do both.